_posts/necessities/2018-02-05-chile-oil.md

```diff
@@ -0,0 +1,35 @@
+
+- Sourcing chilis: I've been on a roll with [Casa Ruiz's arbol chiles](http://casaruizbrand.com/), and I've had good results with [Penzey's Sannam Chilis](https://www.penzeys.com/online-catalog/sanaam-chili-peppers/c-24/p-1256/pd-s) as well, but any dried pepper at all can make a chile oil. 
+
+- Remove stems from dried chiles. While doing this, you can shake out any loose seeds if you'd like a slightly less spicy chile oil. I don't usually bother. 
+
+- Heat a well seasoned cast iron pan on the stove. Do not use any oil. Toast the chiles on both sides until they smell fragrant. Sometimes they blister! That's a good thing! If they burn and blacken, start over, or remove the ones that are burnt. A little bit of burnt pepper is OK, but too much creates an acrid oil that you're not going to enjoy eating. It takes around 2 minutes on each side for the peppers to toast, but watch carefully, peppers vary in toasting time. I usually flip them over individually with chopsticks, a spatula is too coarse of a cooking tool for this job. 
+
+- Remove chiles from the pan as soon as they're toasted. If some are ready sooner than others, remove those first! No rules here except get the chiles toasted and keep 'em from burning. 
+
+- Let the chiles cool a little and then grind them in a cheapo blade grinder. Something like [this](https://www.amazon.com/KRUPS-Electric-Grinder-Stainless-3-Ounce/dp/B00004SPEU/ref=sr_1_4?s=home-garden&ie=UTF8&qid=1517894119&sr=1-4&keywords=spice+grinder) serves nicely if you don't already have one. Of course, you can always use a mortar and pestle but it's really not necessary for this recipe. How fine you grind the peppers is up to you, but enjoy a toothiness to my chile oils--a fine dust is a little too much grinding. 
+
+- Put the ground peppers in a heatproof container (a mason jar is good, a plastic jar is not good. You really don't want this container to break while you're pouring hot oil in. You really, really don't.) that you want to store the chile oil in. 
+
+- Return the pan to the stove. Heat some neutral high heat oil (grapeseed, canola) in the cast iron pan. How much oil? That depends on how much oil vs. chunky pepper mixture you want and how many peppers you toasted. I know, not helpful. But let's say for a 1/4 cup of peppers you'll want at least a half cup of oil. The oil is heated through when a chopstick placed in the oil looks as if the oil is 'bubbling' around it. When you reached this point, pour the oil over your ground chiles. Marvel at the sizzle and smell. Let cool, then cap your container. The chile oil is ready a few hours after made, but tastes better as it ages. It keeps indefinitely at room temperature if you  use clean utensils when serving it.
```

Comment:
- 'neutral, high smoke point oil'
- 'e.g., grapeseed or canola'
- If we agree on the heatproof changes, then this should be amended to include a transfer of the oil and grounds  once cooled.